frappe/docs/user/en/guides/app-development/trigger-event-on-deletion-of-grid-row.md

```diff
@@ -41,3 +41,15 @@ To trigger an event when a row from a Child Table has been deleted (when user cl
       }
    );
  ```
+or
```

Comment:
You can see it is separated by comma and 2nd argument is enclosed in apostrophe.
I tried as suggested as in the original but it does not work, console says something is missing.